src/main/java/com/marklogic/kafka/connect/source/RowBatcherSourceTask.java

```diff
@@ -120,7 +137,7 @@ public synchronized void stop() {
      * @param parsedConfig - The complete configuration object including any transform parameters.
      * @param rowBatcher - The RowBatcher object to be configured.
      */
-    private void configureRowBatcher(Map<String, Object> parsedConfig, RowBatcher<JsonNode> rowBatcher, List<SourceRecord> newSourceRecords) {
+    void configureRowBatcher(Map<String, Object> parsedConfig, RowBatcher<?> rowBatcher) {
```

Comment:
This is fine to keep here. I do think there's some benefit in having RowBatcherBuilder encapsulate all knowledge of how to construct and configure a RowBatcher so that the Task class has no understanding of how that works (I would do that by having a parent AbstractRBB class and tossing this method in there for each subclass to use). For example, we could write small unit tests that invoke an RBB and then inspect the RB that comes back to make sure it's correct. But we can cross that bridge later (if ever), this is fine here.

Reply:
Since I created the AbstractRBB above, I moved this (and others).